### DISCUSSION TITLE
Quasar object is not defined inside a vuex action function

### DISCUSSION THREAD

**Discussion by its-saeed** [quasarframework/quasar#13072] [CLI - BEX mode]
[According to this page][1], I can import `Quasar` object like this outside of a `vue` file: 

    import { Quasar } from 'quasar'
    
    console.log(Quasar.platform.is.ios)

I have this vuex action function:

    import { Quasar } from 'quasar'
    import { getServiceProviderSpec, isPageRegistered } from '../../services/ContentProviderService'
    
    export function updateContentProvider(context) {
      console.log(Quasar.platform.is.ios)
      if (Quasar.platform.is.bex) {
        Quasar.bex.send('bex.get.url').then(url => {
          const spec = getServiceProviderSpec(url.data)
          isPageRegistered(spec.domain, spec.page).then(resp => {
            spec.isRegistered = resp.status === 200
            context.commit('setContentProvider', spec)
          })
        }).catch(e => console.log(e))
      } else {
        // just for test
        context.commit('setCurrentUrl', 'https://instagram.com/s4eed')
      }
    }

What this function does, is not important, but I can't access `Quasar` object here, it's undefined.

  [1]: https://quasar.dev/options/the-q-object#outside-of-a-vue-file

**Comment by BenceSzalai** [comment]
Try maybe with `this.$q.platform.is.ios`...?

**Comment by julie777** [comment]
I think it has changed to 
```
import {useQuasar} from "quasar"

let $q = useQuasar()
```
